dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactoryImpl.java

```diff
@@ -595,4 +595,28 @@ public String suggestVelocityVar(final String tryVar, List<Field> takenFields) t
         "field.validation.variable.already.taken");
 
   }
+
+  public void moveSortOrderForward(int from, int to) throws DotDataException {
+    DotConnect dc = new DotConnect();
+    dc.setSQL(sql.moveSorOrderForward);
+    dc.addParam(from);
+    dc.addParam(to);
+    dc.loadResult();
+  }
+
+  public void moveSortOrderBackward(int from, int to) throws DotDataException {
+    DotConnect dc = new DotConnect();
+    dc.setSQL(sql.moveSorOrderBackward);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)